### PR TITLE
[NG] Feedback phase 2 version 2

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

Fix feedback phase 2 version 2 [isssue](https://docs.google.com/document/d/1zfEbndghE2fmleNgVCWHQ9RLGeW_XDG1Vk7qk6mDsA0/edit?ts=5dde23ef)

- [x] Update about page
- [x] Remove the word `campaigners` from 
- [x] Fix label alignments - fix chart with long labels by displaying y-axis
- [x] More data uploads from NBS
 
Fixes [#170028785](https://www.pivotaltracker.com/story/show/170028785)

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
Screenshot from 2019-11-29 15-59-42

Screenshot from 2019-11-29 16-00-03

Screenshot from 2019-11-29 16-14-42

Screenshot from 2019-11-29 17-31-51

Screenshot from 2019-11-29 17-50-14

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by KhadijaMahanga** [comment]
@DavidLemayian There are changes on nigeria about page, is that cool?

File: hurumap_ng/templates/about.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <p>{{ HURUMAP.name }} provides the general public, particularly infomediaries like journalists and data enthusiasts, 
+            a friendly ‘plug & play’ toolkit for locating interactive data visualisations and embedding same in their storytelling. 
+            The project's underlying data is quality-checked and solely sourced from the National Bureau of Statistics (NBS). </p>
+
+        <p>The project is built on <a href="http://wazimap.readthedocs.org/" target="_blank" rel="noopener">Wazimap</a>, an open source platform by <a href="https://openup.org.za/" target="_blank" rel="noopener">OpenUp</a> and <a href="http://www.mediamonitoringafrica.org/" target="_blank" rel="noopener">Media Monitoring Africa</a> for making census data more understandable.</p>
+
+        <br/>
+        {% include 'blocks/about/manifesto.html' %}
+    </div>
+{% endblock %}
```
Comment (kilemensi) [inline hurumap_ng/templates/about.html]: `/n`

**Review by kilemensi — APPROVED** [review]
:rocket: @KhadijaMahanga 

The y-axis values seem cut off to the left... bit of padding/scaling may be?
Screenshot from 2019-12-02 18-24-14

File: hurumap_ng/templates/about.html (hunk 1 above)
Comment (DavidLemayian) [reply hurumap_ng/templates/about.html]: `\n`?

**Pull request merged**